frontend/app/src/locales/en.json

```diff
@@ -824,7 +824,35 @@
     "title": "Import Data"
   },
   "dashboard": {
-    "title": "Dashboard"
+    "title": "Dashboard",
```

Comment:
Nice tooltips too!

Reply:
They were there, I am just doing the extraction of resources step by step :)